realm/realm-library/src/main/java/io/realm/OsRealmSchema.java

```diff
@@ -64,6 +64,12 @@ public RealmObjectSchema create(String className) {
         }
 
         @Override
+        public RealmObjectSchema createWithPrimaryKeyField(String className, String primaryKeyFieldName,
```

Comment:
Cannot delete it now, it is part of the `RealmSchema` interface. will remove it in OS schema migration PR.